### PR TITLE
Fix cursor for avatar and user names in comments

### PR DESCRIPTION
**Pull request by danxuliu**
When it was on an author row the cursor was shown as a pointer, even if clicking on the author row itself does nothing. Also avatars used the default cursor, even if clicking on them shows the contacts menu (if the avatar is for a different user than the current one). Now the cursor is shown as a pointer on avatar and user names of users different than the current one.

There are still some changes pending based on what is decided in #7254 (right now the cursor used for mentions in messages being composed is the default one; changing it to a pointer would require removing `.comment` from `#commentsTabView .comments .message...` and also adding the `currentUser` class to the appropriate mentions through JavaScript).

@jancborchardt  Besides that maybe the contacts menu should be completely disabled for mentions in messages being composed, as it does not provide too much value there (and, moreover, right now it does not appear in the proper position).

### CODE REVIEW

**Comment by jancborchardt** [comment]
@danxuliu yeah, we can remove the menu for comments being composed! :)

**Comment by danxuliu** [comment]
>  yeah, we can remove the menu for comments being composed! :)

Done! I have also disabled the contacts menu for mentions to the current user in final messages.

Also, after disabling the contacts menu for comments being composed and with some changes I made to the original CSS selectors this pull request no longer depends on what is decided in #7254; either way the contacts menu and the cursor will work as expected :-) (or at least I think so :-P )

With all that this pull request is now ready to be reviewed!

File: apps/comments/js/commentstabview.js [hunk 1]
```diff
@@ -445,9 +454,11 @@
 				+ ' data-user-display-name="'
 				+ _.escape(displayName) + '"></div>';
 
+			var isCurrentUser = (uid === oc_current_user);
```
Comment (nickvergessen) [inline apps/comments/js/commentstabview.js]: oc_current_user is deprecated?

**Review by nickvergessen — APPROVED** [review]
works 👍

File: apps/comments/js/commentstabview.js (hunk 1 above)
Comment (LukasReschke) [reply apps/comments/js/commentstabview.js]: Let me replace that with `OC.getCurrentUser().uid`

**Review by LukasReschke — COMMENTED** [review]
@danxuliu Please check d15ca9ffafcb2f3bdec5e53922fb5b5531a9fea3. THX :)

**Review by blizzz — APPROVED** [review]
tested and works

**Comment by danxuliu** [comment]
(Yes, I am late to the party, but anyway)

Oops, I did not know that `oc_current_user` was deprecated, sorry!

@LukasReschke Your fix was perfect, thanks :-)

File: apps/comments/js/commentstabview.js [hunk 2]
```diff
@@ -395,16 +397,23 @@
...
+			}
+
 			$el.find('.avatar').each(function() {
 				var avatar = $(this);
 				var strong = $(this).next();
 				var appendTo = $(this).parent();
 
-				$.merge(avatar, strong).contactsMenu(avatar.data('user'), 0, appendTo);
+				var username = $(this).data('username');
+				if (username !== oc_current_user) {
```
Comment (nickvergessen) [inline apps/comments/js/commentstabview.js]: :( @danxuliu please adjust here too

**Pull request merged**